airflow/ui/src/pages/DagsList/Dag/Header.tsx

```diff
@@ -59,7 +63,15 @@ export const Header = ({
             />
           )}
         </HStack>
-        <Flex>{dag ? <TriggerDAGTextButton dag={dag} /> : undefined}</Flex>
+        <Flex>
+          {dag ? (
+            <HStack>
+              <DagDocumentationButton setIsDocsOpen={setIsDocsOpen} />
```

Comment:
1. We should only show the docs button if there are docs
2. I think we can move the dag docs button and modal into a single component that manages its own state. Then we have fewer imports and props to pass around.

Reply:
1. Agree on not showing the button when `doc_md` is empty
2. Is it possible to have `Dialog` outside the `tabs.tsx` ? I tried having it as a component and remember facing is not showing up in expected way.

Reply:
Wrap both the button and the Dialog inside of a `<Box>` should solve that I think